vulnerabilities/utils.py

```diff
@@ -410,3 +410,15 @@ def base32_custom(btes):
             + _base32_table[c & 0x3FF]  # bits 21 - 30  # bits 31 - 40
         )
     return bytes(encoded)
+
+
+# 9/28/2022 Wednesday 1:07:41 PM.Copy from /home/jmh/dev/nexb/vulnerablecode/vulnerabilities/importers/alpine_linux.py
```

Comment:
You may want to remove these comment?
Also this is a good thing to have moved this to a utils directory but you should also update alpine linux by removing the function from https://github.com/nexB/vulnerablecode/blob/6bad93197d1926020f143d1bae7ccc3cb96c087d/vulnerabilities/importers/alpine_linux.py#L61 and using this shared utility function. Otherwise you are creating a copy of the code which usually is a technical debt.

Reply:
Thanks for catching this lingering comment @pombredanne -- I intended to remove but forgot -- a good lesson.

And even more valuable -- thank you for your advice re updating alpine linux with the new `utils.py` function that I created from the old alpine linux function!  🙂 Plus, a failed isort check taught me that 

```
from vulnerabilities.utils import fetch_response, is_cve
```

should instead be

```
from vulnerabilities.utils import fetch_response
from vulnerabilities.utils import is_cve
```